### ISSUE TITLE
Installation is super slow , why ?

### ISSUE THREAD

**Issue by eromoe** [imagemin/gifsicle-bin#81]
I see a issue https://github.com/imagemin/gifsicle-bin/issues/67 , it says due to china GFW ? 

I have already changed to a china mirror,   download looks quite quick.  But stuck at 
```
[root@s01 annotator]# npm install
npm WARN lifecycle react-boilerplate@3.4.0~preinstall: cannot run in wd %s %s (wd=%s) react-boilerplate@3.4.0 np
m run npmcheckversion /mnt/annotator
npm WARN prefer global node-gyp@3.6.2 should be installed with -g

> node-sass@4.6.0 install /mnt/annotator/node_modules/node-sass
> node scripts/install.js

Cached binary found at /root/.npm/node-sass/4.6.0/linux-x64-48_binding.node

> pre-commit@1.2.2 install /mnt/annotator/node_modules/pre-commit
> node install.js

pre-commit:
pre-commit: Detected an existing git pre-commit hook
pre-commit: Old pre-commit hook backuped to pre-commit.old
pre-commit:

> spawn-sync@1.0.15 postinstall /mnt/annotator/node_modules/spawn-sync
> node postinstall


> gifsicle@3.0.4 postinstall /mnt/annotator/node_modules/gifsicle
> node lib/install.js                                   
                                                                        
```

for over one hour.

What's wrong with `node lib/install.js`  ??? 

Does it download some external dependencies ???  If so,  can you point it out , so I can download those deps manually by proxy.

**Comment by eromoe** [comment]
OK， this is the problem
```
var builder = new BinBuild()
        .src('https://github.com/kohler/gifsicle/archive/v1.88.tar.gz')
        .cmd('autoreconf -ivf')
        .cmd(cfg)
        .cmd('make install');
```

But , how do I modify this code made it support a already downloaded `v1.88.tar.gz'`  ??

**Comment by b1f6c1c4** [comment]
In the [`index/js`](https://github.com/imagemin/gifsicle-bin/blob/7f88f03e5def54531cf6f25db53e4e2da2cd8c4b/lib/index.js#L6), the `BinWrapper` tries to download the binary from raw.githubusercontent.com.

I came across exactly the same issue for the first time. However, I the problem went away after I downloaded that file using Chrome. I suspect that this is related to GFW, but I have no idea how this issue begins.

A possible workaround is to manually download the required file and place it at `path.join(__dirname, '../vendor')`. @eromoe
